tracer/test/snapshots/Iast.TrustBoundaryViolation.AspNetCore5.IastEnabled.RedactionEnabled.verified.txt

```diff
@@ -0,0 +1,124 @@
+﻿[
+  {
+    TraceId: Id_1,
+    SpanId: Id_2,
+    Name: aspnet_core.request,
+    Resource: GET /iast/tbv,
+    Service: Samples.Security.AspNetCore5,
+    Type: web,
+    Tags: {
+      aspnet_core.endpoint: Samples.Security.AspNetCore5.Controllers.IastController.Tbv (Samples.Security.AspNetCore5),
+      aspnet_core.route: iast/tbv,
+      component: aspnet_core,
+      env: integration_tests,
+      http.method: GET,
+      http.request.headers.host: localhost:00000,
+      http.route: iast/tbv,
+      http.status_code: 200,
+      http.url: http://localhost:00000/Iast/Tbv?name=name&value=value,
+      http.useragent: Mistake Not...,
+      language: dotnet,
+      runtime-id: Guid_1,
+      span.kind: server,
+      _dd.git.commit.sha: aaaaaaaaaaaaaaaaaaaaabbbbbbbbbbbbbbbbbbbbbbb,
+      _dd.git.repository_url: https://github.com/DataDog/dd-trace-dotnet,
+      _dd.iast.enabled: 1,
+      _dd.iast.json:
+{
+  "vulnerabilities": [
+    {
+      "type": "TRUST_BOUNDARY_VIOLATION",
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "name",
+            "source": 0
+          }
+        ]
+      }
+    },
+    {
+      "type": "TRUST_BOUNDARY_VIOLATION",
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "value",
+            "source": 1
+          }
+        ]
+      }
+    },
+    {
+      "type": "TRUST_BOUNDARY_VIOLATION",
+      "evidence": {
+        "valueParts": [
+          {
+            "value": "name",
+            "source": 0
+          },
+          {
+            "value": "-"
+          },
+          {
+            "value": "value",
+            "source": 1
+          }
+        ]
+      }
+    },
```

Comment:
Is it expected to have these duplicated?

Reply:
Yes, indeed, different session methods, different vulns